library/discovery_api_k8s.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
it might be the case that the `env` binary is not present in the container

Reply:
Probably, but also all other modules of ansible just use `/usr/bin/python`.